.github/workflows/main.yml

```diff
@@ -19,36 +19,30 @@ jobs:
       matrix:
         include:
           # MANDATORY CHECKS USING CURRENT DEVELOPMENT INTERPRETER
-          - python-version: 3.8.6
-            dependencies: >
-              dbus-python==1.2.16
-              pylint==2.4.4
-              psutil==5.6.7
+          - dependencies: >
+              dbus-python-devel
+              pylint
+              python3-psutil
+            image: fedora:33
```

Comment:
Set back to fedora:33, we definitely need to work through w/ the correct values.

Reply:
It says `fedora:33` right now, in the "make lint" task.  Should it say something else?

Reply:
> It says `fedora:33` right now, in the "make lint" task. Should it say something else?

Whoops! I meant "fedora:32".